CMakeLists.txt

```diff
@@ -513,11 +516,20 @@ if(NOT HEADLESS)
   if (APPLE AND EXISTS /usr/local/opt/qt)
     list(APPEND CMAKE_PREFIX_PATH "/usr/local/opt/qt")
   endif()
-  if (APPLE AND EXISTS /usr/local/opt/qt@5)
-    list(APPEND CMAKE_PREFIX_PATH "/usr/local/opt/qt@5")
-  endif()
-  if (APPLE AND EXISTS /opt/homebrew/opt/qt@5)
-    list(APPEND CMAKE_PREFIX_PATH "/opt/homebrew/opt/qt@5")
+  if (USE_QT6)
+    if (APPLE AND EXISTS /usr/local/opt/qt@6)
+      list(APPEND CMAKE_PREFIX_PATH "/usr/local/opt/qt@6")
+    endif()
+    if (APPLE AND EXISTS /opt/homebrew/opt/qt@6)
+      list(APPEND CMAKE_PREFIX_PATH "/opt/homebrew/opt/qt@6")
+    endif()
+  else()
+    if (APPLE AND EXISTS /usr/local/opt/qt@5)
+      list(APPEND CMAKE_PREFIX_PATH "/usr/local/opt/qt@5")
+    endif()
+    if (APPLE AND EXISTS /opt/homebrew/opt/qt@5)
+      list(APPEND CMAKE_PREFIX_PATH "/opt/homebrew/opt/qt@5")
+    endif()
```

Comment:
This is getting a bit messy - do you think there's a way of moving this out of the main CMakeLists file?

Same goes for the Qt5 vs. Qt6 logic. Perhaps a separate macro which deals with the details without making the main CMakeLists file so messy?

Reply:
ok, let's defer this until we have both Qt5+6 working and have a better picture